Lesson24/gokce/styles.css

```
@@ -0,0 +1,148 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: "Poppins", sans-serif;
+}
+.nav-container {
+  width: 100%;
+  height: 60px;
+  background-color: #333;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+}
+.latest-text {
+  text-align: center;
+  margin-top: 20px;
+  font-size: 1.8em;
+  color: #333;
+}
+
+.nav-container ul {
+  list-style: none;
+  display: flex;
+  gap: 50px;
+
+  justify-content: space-around;
+}
+
+header {
+  background-color: #f4f4f4;
+  padding: 20px 0;
+  text-align: center;
+}
+#toggle-nav {
+  display: none;
+}
+
+.nav-container li a {
+  color: #ffffff;
+  text-decoration: none;
+}
+
+.post-content {
+  width: 100%;
+  margin: 0;
+  padding: 0 20px;
+}
+.post-card {
+  border: 1px solid #e0e0e0;
+  border-radius: 8px;
+  padding: 20px;
+  box-shadow: 0 2px 5px #0000000d;
+  max-width: 700px;
+  margin: 20px auto;
+}
+
+.post-title {
+  font-size: 2.5em;
+  margin-bottom: 10px;
+}
+.post-body {
+  font-size: 1.2em;
+  line-height: 1.6;
+}
+
+.post-content .delete-btn {
+  background-color: #e74c3c;
+  color: #ffffff;
+  padding: 10px 20px;
+  margin: 10px 0;
+  border: none;
+  border-radius: 5px;
+  cursor: pointer;
+}
+
+.deleted {
+  display: none;
+}
+
+footer {
+  background-color: #333;
+  color: #ffffff;
+  text-align: center;
+  padding: 10px 0;
+
+  width: 100%;
+}
+
+@media screen and (max-width: 600px) {
+  .nav-container ul {
+    display: none;
+    flex-direction: column;
+    gap: 20px;
+    align-items: center;
+  }
+  .nav-container li {
+    border-bottom: 1px solid #ffffff;
+    width: 100%;
+    height: 40px;
```

Accessibility guidelines (WCAG) recommend a minimum touch target size of `44px` (or `48px`) so users with larger fingers can tap links easily without mistakes. Increasing this slightly improves the mobile experience.

```suggestion
    height: 48px;
```